ntosebpfext.sln

```diff
@@ -108,172 +108,358 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "include", "include", "{6F4A
 		include\ebpf_ntos_program_attach_type_guids.h = include\ebpf_ntos_program_attach_type_guids.h
 	EndProjectSection
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "onebranch", "tools\onebranch\onebranch.vcxproj", "{988A1731-F674-472F-8063-84D6DEA8988D}"
+	ProjectSection(ProjectDependencies) = postProject
+		{1132BFD8-EECA-4372-B08E-D349D86FE616} = {1132BFD8-EECA-4372-B08E-D349D86FE616}
+		{231EE32B-EBA4-4FE5-A55B-DB18F539D403} = {231EE32B-EBA4-4FE5-A55B-DB18F539D403}
+		{55499E36-37D4-4F86-B694-9F2990315758} = {55499E36-37D4-4F86-B694-9F2990315758}
+		{61896FC2-D314-468E-887B-7B1EBC7F0FEB} = {61896FC2-D314-468E-887B-7B1EBC7F0FEB}
+		{DB2CE6D8-F3B3-4C35-A585-5C88B4684821} = {DB2CE6D8-F3B3-4C35-A585-5C88B4684821}
+		{FA9BB88D-8259-40C1-9422-BDEDF9E9CE68} = {FA9BB88D-8259-40C1-9422-BDEDF9E9CE68}
+	EndProjectSection
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|Any CPU = Release|Any CPU
 		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
+		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{231EE32B-EBA4-4FE5-A55B-DB18F539D403}.Debug|Any CPU.ActiveCfg = Debug|x64
```

Comment:
>|Any CPU [](http://example.com/codeflow?start=46&length=8)

I've been trying to avoid adding "Any CPU" to the build here since the managed code should always know what bitness it's on for how we marshal pointers and such - so I think you may be able to revert most of the changes to the `.sln` file or go into the VS configuration manager and delete the "Any CPU" configuration to remove most of the diff here

Reply:
I also propose we delete x86, since we don't ship a 32-bit OS anymore (as of Windows 11)

Reply:
Fixed.